src/auth/service/auth.service.ts

```diff
@@ -131,6 +131,7 @@ export class AuthService {
   }
 
   async resendEmailVerificationCode(email: string) {
+    email = email.toLowerCase();
```

Comment:
Could we just use the `transform` property from the DTO every time we receive an email (maybe except for linkedin auth where we can just pass it as a lowercased string from the controller). This way we don't have to modify a property in place

Reply:
I think I'll just add a `LowercaseEmail` class with just the email property. That should do it.